source/tutorial/react-native.txt

```diff
@@ -13,14 +13,24 @@ React Native Tutorial
 Overview
 --------
 
-You can use the Realm JavaScript SDK, and the official Realm React
-library to build a mobile application with React Native. This tutorial
+You can use the Realm React Native SDK and :npm:`@realm/react`
+to build a mobile application with React Native. This tutorial
 walks you through how to build your own app that uses Flexible Sync.
 
-For this tutorial, we'll start with a pre-built template application to
-see how everything fits together. Then, you will add a new Priority
-field to the existing Item model and update the Flexible Sync
-Subscription to only show items within a range of priorities.
+For this tutorial, we'll start with a pre-built TypeScript template application
+to see how everything fits together. 
+
+After you've got the template app running, you will add a new ``priority``
+field to the existing ``Item`` model and update the Flexible Sync
+subscription to only show items within a range of priorities. This example 
+illustrates how you might adapt the template app for your own needs. 
+You would not necessarily make this change given the current structure of 
```

Comment:
What does this mean?

Reply:
I pulled this language from the [SwiftUI v2 template tutorial PR](https://github.com/mongodb/docs-app-services/pull/276/files#diff-d9f63f6fb3aa07493b5e55a88e6647bedfded0df4a8847c03e2b398518a82db6R35-R37). I think the idea is to communicate that there's likely a better way to handle this for a production app.